pkgs/by-name/co/cosmic-icons/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, just
+, pop-icon-theme
+, hicolor-icon-theme
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "cosmic-icons";
+  version = "unstable-2023-08-30";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = "14d8e2048087be1ad444f9b3ebb75885509f72c6";
+    sha256 = "sha256-WbdgHmTn403x95x9wEYL0T9ksbN+YLzEB2yE0UrF9T0=";
+  };
+
+  nativeBuildInputs = [ just ];
+
+  justFlags = [
+    "--set"
+    "prefix"
+    (placeholder "out")
+  ];
+
+  propagatedBuildInputs = [
+    pop-icon-theme
+    hicolor-icon-theme
```

Comment:
Are these actually necessary? How are they getting picked up?

Reply:
Yes, since they are propagated, they create symlinks in `share`.
The `cosmic-icons` inherit from both of these icon sets: https://github.com/pop-os/cosmic-icons/blob/14d8e2048087be1ad444f9b3ebb75885509f72c6/index.theme#L5